projects/element-ng/tabs/si-tabset.component.ts

```diff
@@ -68,6 +69,21 @@ export class SiTabsetComponent {
     return tab instanceof SiTabLinkComponent;
   }
 
+  constructor() {
+    effect(() => {
+      if (this.activeTab()?.active() && this.showMenuButton()) {
```

Comment:
```suggestion
      if (this.showMenuButton() && this.activeTab()) {
```

I think there is no need to check if the activeTab is active.
Switiching the order should ensure that the effect is triggered less often